beetmoverscript/newsfragments/124.added

```diff
@@ -0,0 +1 @@
+Add automatic changelog generation using [towncrier](https://github.com/hawkowl/towncrier/).
```

Comment:
Do we add these newsfragments manually? What do we name them if there's no related issue?